torch/fx/experimental/symbolic_shapes.py

```diff
@@ -446,8 +499,8 @@ def _maybe_guard_eq(self, expr: "sympy.Eq") -> None:
             if expr.has(sympy.Mod):
                 mod_expr = tuple(expr.atoms(sympy.Mod))[0]
                 try:
-                    solutions = sympy.solve(lhs - rhs, mod_expr, dict=True)
-                    if len(solutions) == 1 and solutions[0][mod_expr] == 0:
+                    solutions = sympy.solve(lhs - rhs, mod_expr)
```

Comment:
Don't land this, I think.

Reply:
I'm going to revert this from branch too then